### PR TITLE
Adds documentation outlining vote configuration

### PR DESCRIPTION
**Pull request by jthiller**
Takes a first pass at documenting [Helium Vote](https://helium.vote) and some details around the configuration of a vote.

For visibility cc/ @abhay @cokes518 

Edit: Quick preview link https://deploy-preview-706--helium-docs.netlify.app/community-voting

### CODE REVIEW

**Comment by cokes518** [comment]
nice guide @jthiller. One thing I noticed is that "Voting Power" is specially based off HIP39 voting and may not apply to future votes. This "Voting Power" may not be token-based in future votes, depending on the issue at hand. For example, a vote on AU915 may be number of Hotspots owned by a voting wallet that is asserted in Australia. 

I'd recommend detailing that Voting Power may change vote-to-vote, depending on what the community is reaching consensus on.

**Comment by jthiller** [comment]
fantastic! Thanks I'll get that noted.

**Comment by jthiller** [comment]
This is ready for review/merge.

File: docs/community-governance/community-voting.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+# Voting on Helium
+In order to approve or deny changes to the blockchain, economics, or governance of the Helium Network; Helium and DeWi leverage on-chain voting which is open to the entire community. This on-chain voting is currently facilited through the website [helium.vote](https://helium.vote). 
+
+For most votes, voting power is currently determined by the HNT balance of the voting wallets at a rate of one vote per HNT. However, voting power may change vote-to-vote, depending on what the community is reaching consensus on. As example, a vote on AU915 may only concern Hotspots in Australia and use Hotspots as votes. Proposals to implement more involved voting mechanisms have been presented, but have not been voted on or implemented yet. See: [HIP41](https://github.com/helium/HIP/blob/master/0041-governance-by-token-lock-v2.md)
```
Comment (lthiery) [inline docs/community-governance/community-voting.mdx]:
```suggestion
For most votes, voting power is currently determined by the HNT balance of the voting wallets at a rate of one vote per HNT. However, voting power may change vote-to-vote, depending on what the community is reaching consensus on. As example, a vote on on frequency plan for a region would only concern Hotspots in the region and one would use number Hotspots in the region as votes. Proposals to implement more involved voting mechanisms have been presented, but have not been voted on or implemented yet. See: [HIP41](https://github.com/helium/HIP/blob/master/0041-governance-by-token-lock-v2.md)
```

**Review by lthiery — CHANGES_REQUESTED** [review]
I prefer a more general statement around the hotspot voting.

**Review by lthiery — APPROVED** [review]

**Pull request merged**